packages/idyll-components/test/components.js

```diff
@@ -4,82 +4,256 @@ import Adapter from 'enzyme-adapter-react-16';
+
+  describe('Full Mount', () => {
+    describe('Layout', () => {
+      it('<Aside />', () => {
+        const wrapper = mount(<Aside />);
+      });
+      it('<FeatureContent />', () => {
+        const wrapper = mount(<FeatureContent />);
+      });
+      it('<Feature />', () => {
+        const wrapper = mount(<Feature />);
+      });
+      it('<Fixed />', () => {
+        const wrapper = mount(<Fixed />);
+      });
+      it('<Float />', () => {
+        const wrapper = mount(<Float />);
+      });
+      it('<FullScreen />', () => {
+        const wrapper = mount(<FullScreen />);
+      });
+      it('<Inline />', () => {
+        const wrapper = mount(<Inline />);
+      });
+      it('<Panel />', () => {
+        const wrapper = mount(<Panel />);
+      });
+      it('<Waypoint />', () => {
+        const wrapper = mount(<Waypoint />);
+      });
+    })
+
+    describe('Presentation', () => {
+      it('<Action />', () => {
+        const wrapper = mount(<Action />);
+      });
+      it('<Boolean />', () => {
+        const wrapper = mount(<Boolean />);
+      });
+      it('<Button />', () => {
+        const wrapper = mount(<Button />);
+      });
+      it('<Chart />', () => {
+        const wrapper = mount(<Chart />);
+      });
+      it('<Display />', () => {
+        const wrapper = mount(<Display />);
+      });
+      it('<Dynamic />', () => {
+        const wrapper = mount(<Dynamic />);
+      });
+      it('<Equation />', () => {
+        const wrapper = mount(<Equation />);
+      });
+      it('<Gist />', () => {
+        const wrapper = mount(<Gist />);
+      });
+      it('<Header />', () => {
+        const wrapper = mount(<Header />);
+      });
+      it('<Link />', () => {
+        const wrapper = mount(<Link />);
+      });
+      it('<Radio />', () => {
+        const wrapper = mount(<Radio />);
+      });
+      it('<Range />', () => {
+        const wrapper = mount(<Range />);
+      });
+      it('<Select />', () => {
+        const wrapper = mount(<Select />);
+      });
+      it('<Slide />', () => {
+        const wrapper = mount(<Slide />);
+      });
+      it('<Slideshow />', () => {
+        const wrapper = mount(<Slideshow />);
+      });
+      it('<SVG />', () => {
+        const wrapper = mount(<SVG />);
+      });
+      it('<Table />', () => {
+        const wrapper = mount(<Table />);
+      });
+      it('<TextInput />', () => {
+        const wrapper = mount(<TextInput />);
+      });
+    });
+
+    describe('Helpers', () => {
+      it('<Analytics />', () => {
+        const wrapper = mount(<Analytics />);
+      });
+      it('<CodeHighlight />', () => {
+        const wrapper = mount(<CodeHighlight />);
+      });
+      it('<Container />', () => {
+        const wrapper = mount(<Container />);
+      });
+      it('<Preload />', () => {
+        const wrapper = mount(<Preload />);
+      });
+      it('<Screen />', () => {
+        const wrapper = mount(<Screen />);
+      });
+    });
```

Comment:
Nothing is being asserted in this file... If we're just making sure they don't blow up maybe we should use `expect(shallow(<Screen />)).toNotThrow()` or whatever?